tests/data/test_fields.py

```diff
@@ -0,0 +1,197 @@
+import pytest
+from datetime import datetime
+from enum import Enum
+from uuid import uuid4
+
+from openapi.data.fields import (
+    ValidationError, data_field, bool_field, uuid_field, number_field,
+    decimal_field, email_field, enum_field, date_time_field,  # ListValidator,
+    UUIDValidator, EnumValidator, Choice, DateTimeValidator, NumberValidator,
+    DecimalValidator, email_validator, BoolValidator
+)
+
+
+class FakeEnum(Enum):
+    VALUE_A = 1
+    VALUE_B = 2
+    VALUE_C = 3
+
+
+def test_ListValidator():
+    pass
+
+
+def test_UUIDValidator_call_valid():
+    uuid = uuid4()
+    field = uuid_field()
+    validator = UUIDValidator()
+    assert validator(field, uuid) == uuid.hex
+    assert validator(field, uuid.hex) == uuid.hex
+
+
+def test_UUIDValidator_call_invalid():
+    field = uuid_field()
+    validator = UUIDValidator()
+    with pytest.raises(ValidationError):
+        validator(field, 'id')
+
+
+def test_UUIDValidator_dump():
+    value = uuid4()
+    validator = UUIDValidator()
+    assert validator.dump(value) == value.hex
+    assert validator.dump(value.hex) == value.hex
+
+
+def test_EnumValidator_call_valid():
+    field = enum_field(FakeEnum)
+    validator = EnumValidator(FakeEnum)
+    assert validator(field, None) is None
```

Comment:
Not directly related to the tests, but I am not sure why we would not raise in this case

Reply:
I had to allow `None` on this validator before for some reason, but I don't know if it's necessary anymore. I'll check if we can raise on this case